wlog/svclog/svc1log/params.go

```diff
@@ -106,6 +106,14 @@ func OriginFromInitPkg(skipPkg int) Param {
 	return Origin(CallerPkg(1, 0))
 }
 
+var defaultOriginFromCallLineStackSkip = 8
+
+// SetOriginFromCallLineStackSkip sets the number of frames that OriginFromCallLine should skip to identify the filename
+// and line of the logger invocation.
+func SetOriginFromCallLineStackSkip(skip int) {
```

Comment:
Would there ever be a case where you'd want less than 8 frames? It might be more intuitive if this configures "additional" frames and we add it to the default tracked in this package. That way if we were to refactor code in here which changes that default, calling code would continue to work.

Reply:
Good question @bmoylan  -- I think it would be possible if someone implements another svclogger that doesn't wrap the default logger. 

If that needs to happen though, I'd advocate for the approach outlined in the `Possible downsides?` section in the PR header

Reply:
@nmiyake do you have thoughts on this?

Reply:
I don't think we want this value to be global, but instead stored on each logger since different implementations could have different skip counts. I also worry about the thing I flagged above, where seemingly innocuous changes to this library will break people who've hardcoded a number.

Adding a setter to the interface for every implementation to handle feels overkill when the origin param is the only place it's used. I'm thinking this config is best provided as an argument to the OriginFromCallLine() param where it is relevant since nothing else cares. I don't want a breaking change, but maybe we can do something like

```go
func OriginFromCallLine() Param {
    return OriginFromCallLineWithSkip(0)
}

const defaultOriginFromCallLineStackSkip = 8

func OriginFromCallLineWithSkip(int skipFrames) Param {
	return paramFunc(func(entry wlog.LogEntry) {
		origin := ""
		if file, line, ok := initLineCaller(defaultOriginFromCallLineStackSkip + skip); ok {
			origin = file + ":" + strconv.Itoa(line)
		}
		entry.OptionalStringValue(OriginKey, origin)
	})
}
```

Would that work?

Reply:
I opened https://github.com/palantir/witchcraft-go-logging/pull/107 and https://github.com/palantir/witchcraft-go-server/pull/215 to demo this approach